mongonaut/views.py

```diff
@@ -98,7 +96,7 @@ def get_queryset(self):
             self.page = 1
 
         obj_count = queryset.count()
-        self.total_pages = math.ceil(obj_count / self.documents_per_page)
+        self.total_pages = obj_count / self.documents_per_page + (1 if obj_count % self.documents_per_page else 0)
```

Comment:
Why the mathematical change here? What was broken before? This is much harder to read and understand.